splunk-otel-android/src/main/java/com/splunk/rum/SplunkRum.java

```diff
@@ -67,6 +67,9 @@ public class SplunkRum {
 
     static final AttributeKey<String> APP_NAME_KEY = stringKey("app");
     static final AttributeKey<String> RUM_VERSION_KEY = stringKey("splunk.rum.version");
+    static final AttributeKey<String> APPLICATION_ID_KEY = stringKey("app.application.id");
+    static final AttributeKey<String> APP_VERSION_CODE_KEY = stringKey("app.version.code");
+    static final AttributeKey<String> SPLUNK_OLLY_UUID_KEY = stringKey("app.splunk.olly.uuid");
```

Comment:
I'm sure I've said it before, but it would be lovely to see these added to the official otel semconv.